doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -464,6 +470,92 @@ disabled, the liveness check uses `GET localhost:8080/ping`
 *  `vertx.health.path` set to "/ping" and `vertx.health.readiness.path` set to "/ready"  => the readiness check uses
 `GET localhost:8080/ready` , the liveness check uses the `GET localhost:8080/ping`
 
+[[f8-maven-scm-enricher]]
+==== f8-maven-scm-enricher
+
+This enricher adds additional SCM related metadata to deployments and services.  These metadata will be added only if
+`<scm>` information is present in the maven `pom.xml` of the project.
+
+The following annotations will be added to the kubernetes deployment and service
+
+.Maven SCM Enrichers Annotation Mapping
+[cols="2,2"]
+|===
+| Maven SCM Info | Annotation
+
+| scm/connection
+| fabric8.io/scm-con-url
+
+| scm/developerConnection
+| fabric8.io/scm-devcon-url
+
+| scm/tag
+| fabric8.io/scm-tag
+
+| scm/url
+| fabric8.io/scm-url
+
+|===
+
+Lets say you have a maven pom.xml with the following scm information,
+
+[source,xml]
+
+<scm>
+    <connection>scm:git:git://github.com/fabric8io/fabric8-maven-plugin.git</connection>
+    <developerConnection>scm:git:git://github.com/fabric8io/fabric8-maven-plugin.git</developerConnection>
+    <url>git://github.com/fabric8io/fabric8-maven-plugin.git</url>
+</scm>
+
+This infomation will be enriched as annotations in the generated manifest like,
+
+[source,yaml]
+...
+fabric8.io/scm-con-url: "scm:git:git://github.com/fabric8io/fabric8-maven-plugin.git"
```

Comment:
maybe one could add the `annotations:` level (and some upper level, too. Maybe up to the top for a `Deployment`'s annotation ?) to provide more context in this (and the other example).